client/src/screens/BallotScreen.tsx

```diff
@@ -47,31 +52,34 @@ const BallotScreen = () => {
           </Monospace>
         </p>
         <HorizontalRule />
-        <p><strong>Ballot style {ballotStyle.id}</strong> has the following <strong>{pluralize('contest', ballotContests.length, true)}</strong>.</p>
-        {ballotContests
-          .map(contest => (
-            <React.Fragment key={contest.id}>
-              <h3>{contest.title}</h3>
-              {contest.type === 'candidate' ? (
-                <ul>
-                  {contest.candidates.map(candidate => (
-                    <li key={candidate.id}>{candidate.name}</li>
-                  ))}
-                </ul>
-              ) : (
-                  <ul>
-                    <li>Yes</li>
-                    <li>No</li>
-                  </ul>
-                )}
-            </React.Fragment>
-          ))}
+        <p>
+          <strong>Ballot style {ballotStyle.id}</strong> has the following{' '}
+          <strong>{pluralize('contest', ballotContests.length, true)}</strong>.
+        </p>
+        {ballotContests.map((contest) => (
+          <React.Fragment key={contest.id}>
+            <h3>{contest.title}</h3>
+            {contest.type === 'candidate' ? (
+              <ul>
+                {contest.candidates.map((candidate) => (
+                  <li key={candidate.id}>{candidate.name}</li>
+                ))}
+              </ul>
+            ) : (
+              <ul>
+                <li>Yes</li>
+                <li>No</li>
+              </ul>
+            )}
+          </React.Fragment>
+        ))}
         <HorizontalRule />
       </NavigationScreen>
       <HandMarkedPaperBallot
         ballotStyleId={ballotStyleId}
         election={election}
         precinctId={precinctId}
+        dualLanguageCode={languageCode}
```

Comment:
nit: should this be named `secondLanguageCode` rather than `dual` ?

Reply:
👍

Reply:
Force pushed this change.